.github/workflows/build_and_push.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Build and Push Container to ECR, deploy to lambda
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [master]
+
+env:
+  GITHUB_SHA: ${{ github.sha }}
+  REGISTRY: ${{ secrets.AWS_ACCOUNT }}.dkr.ecr.ca-central-1.amazonaws.com/notify/api
```

Comment:
i would leave this a `.../notify` because on line 73 you are adding a name for the image